pkg/chain/gen/celo/Makefile

```diff
@@ -16,6 +16,10 @@ contract_files := $(addprefix contract/,$(addsuffix .go,$(subst ImplV1,,$(clean_
 abi_files := $(addprefix abi/,$(addsuffix .abi,$(clean_contract_stems)))
 abigen_files := $(addprefix abi/,$(addsuffix .go,$(clean_contract_stems)))
 
+# Additional build tags which should be passed while running `abigen` command.
+ABIGEN_BUILD_TAGS ?= default
```

Comment:
What is `default`, where does it come from?

Reply:
It's just an arbitrary value indicating the default value of `ABIGEN_BUILD_TAGS` variable is in use. I felt this was more verbose than just empty quotes.

Reply:
And if we drop this line entirely and `ABIGEN_BUILD_TAGS` are not set, what is going to happen?

Reply:
`go run` will fail as it will run with an empty build tag list.

Reply:
I see. Can we add a comment explaining what `default` is and why is it needed?

Reply:
Sure, please see https://github.com/keep-network/keep-ecdsa/commit/0f23d1f7f7937505a064915e37bef215a583f6cf